src/EcsRx.Tests/Framework/EntityTests.cs

```diff
@@ -212,5 +212,25 @@ public void should_add_components_in_parameter_order()
                 componentDatabase.Set(2, 1, fakeComponents[2]);
             });
         }
+
+        [Fact]
+        public void should_entity_components_returns_not_one_same_component()
```

Comment:
Can we rename this to something more generic as the test is really just confirming all components are returned correctly
```suggestion
        public void should_return_all_components_allocated_to_the_entity()
```

Reply:
yes we can